src/main/java/org/example/homework7/RomanNumbersCalculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.homework7;
+
+public class RomanNumbersCalculator {
```

Comment:
Consider moving all classes related to Roman Number Calculator task to separate package inside homework7 package